Tests/FluentTests/ChildTests.swift

```diff
@@ -0,0 +1,112 @@
+import XCTest
+import Fluent
+
+class ChildTests: XCTestCase {
+    
+    static let allTests = [
+        ("testChildRelations", testChildRelations)
+    ]
+    
+    func testChildRelations() throws {
+        let memoryDriver = MemoryDriver()
+        let database = Database(memoryDriver)
+        
+        try database.prepare(Owner.self)
+        try database.prepare(Pet.self)
+        
+        var owner1 = Owner(name: "John")
+        try owner1.save()
+        
+        var pet1 = Pet(name: "Fodo", owner: owner1)
+        try pet1.save()
+        
+        let petsOwner = try pet1.getOwner()
+        
+        XCTAssertNotNil(petsOwner)
+        
+        let pets = try owner1.getPets()
+        
+        XCTAssertNotEqual(0, pets.count)
+    }
+    
+}
+
+class Owner: Entity {
+    var id: Node?
+    var exists: Bool = false
+    
+    var name: String
+    
+    init(name: String) {
+        self.name = name
+    }
+    
+    required init(node: Node, in context: Context) throws {
+        id = try node.extract("id")
+        name = try node.extract("name")
+    }
+    
+    static func prepare(_ database: Fluent.Database) throws {
+        try database.create("Owners") { pet in
```

Comment:
lets try to always use `create(entity)` users get these two mixed up so we're trying to make internal code consistent.